paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/LogSystemITCase.java

```diff
@@ -186,4 +190,93 @@ public void testReadFromLogWithOutSteamingReadMode() throws Exception {
                 .hasMessage(
                         "File store continuous reading does not support the log streaming read mode.");
     }
+
+    @Test
+    @Timeout(60)
+    public void testLogSystemAutoRegister() throws TableNotExistException {
+        // enable log system auto registration
+        tEnv.executeSql(
+                String.format(
+                        "CREATE CATALOG PAIMON_REGISTER WITH ("
+                                + "'type'='paimon', 'warehouse'='%s', 'log.system.auto-register'='true')",
+                        getTempDirPath()));
+        tEnv.useCatalog("PAIMON_REGISTER");
+
+        env.getCheckpointConfig().disableCheckpointing();
+        env.setParallelism(1);
+        tEnv.executeSql(
+                String.format(
+                        "CREATE TABLE T (i INT, j INT) WITH ("
+                                + "'log.system'='kafka', "
+                                + "'log.system.partitions'='2', "
+                                + "'write-mode'='append-only', "
+                                + "'kafka.bootstrap.servers'='%s', "
+                                + "'kafka.topic'='T')",
```

Comment:
Should we need to throw exception when user configure the topic with `auto-register=true` or just use the given topic name?

Reply:
Hi, @FangYongs , I think we should use the given topic name once users want to manage kafka topics by self.